app/controllers/api/v2/courses_controller.rb

```diff
@@ -289,7 +289,10 @@ def enrichment_params
                   :course_code,
                   :subjects_ids,
                   :subjects_types,
-                  :level)
+                  :level,
+                  :degree_grade,
+                  :subject_requirements,
```

Comment:
can we call it 
```suggestion
                  :degree_subject_requirements,
```

since we already have a `subjects` on a `course` as well as `gcse_subjects_required`

Reply:
No probs boss